diff --git a/plat/imx/imx8m/imx8m_psci_common.c b/plat/imx/imx8m/imx8m_psci_common.c
index 4a7255657..71678e955 100644
--- a/plat/imx/imx8m/imx8m_psci_common.c
+++ b/plat/imx/imx8m/imx8m_psci_common.c
@@ -251,12 +251,11 @@ void __dead2 imx_system_off(void)
 	uint32_t val;
 
 	val = mmio_read_32(IMX_SNVS_BASE + SNVS_LPCR);
-	val |= SNVS_LPCR_SRTC_ENV | SNVS_LPCR_DP_EN | SNVS_LPCR_TOP |
-		SNVS_LPCR_LPTA_EN | SNVS_LPCR_LPWUI_EN;
+	val |= SNVS_LPCR_SRTC_ENV | SNVS_LPCR_DP_EN | SNVS_LPCR_TOP;
 	mmio_write_32(IMX_SNVS_BASE + SNVS_LPCR, val);
-
+	INFO("poweroff\n");
 	while (1)
-		;
+	;
 }
 
 void __dead2 imx_pwr_domain_pwr_down_wfi(const psci_power_state_t *target_state)
